src/asciidoc/docs/refs/glossary.adoc

```diff
@@ -0,0 +1,33 @@
+= Glossary
+:page-slug: glossary
+:page-description: Definition of terms
```

Comment:
Let's expand this description a little bit and also add punctuation. The "unfurled" link will show this description when the link is shared.

Reply:
Fixed.